README.md

```diff
@@ -0,0 +1,61 @@
+# terraform-aws-cloudtrail
+
+A Terraform module to create an Amazon Web Services (AWS) CloudTrail trail.
+
+## Usage
+
+This module creates a trail that logs to an S3 bucket. The module can be
+configured to log to an existing S3 bucket, or to make a new one for you
+automatically.
+
+```hcl
+module "cloudtrail" {
```

Comment:
Given that most people will likely copy-and-paste this to get started, I think it would be good to populate it with most/all of the parameters (even if with their default values).

Reply:
Good call! I added in all variables with default values and double-checked to make sure that they work.